client/views/windows/sign_in.tpl

```diff
@@ -1,7 +1,8 @@
 <form class="container" method="post" action="">
 	<div class="row">
-		<div class="col-xs-12">
-			<h1 class="title">Sign in to The Lounge</h1>
+		<div class="text-center">
```

Comment:
Can you set `text-align: center` in CSS instead of using this class please.

Reply:
Yeah, that was dumb...
I did better than that actually, I got rid of the Bootstrap crap and leveraged your flex layout as much as I could 👌